src/node_internals.h

```diff
@@ -121,6 +121,11 @@ inline v8::Local<TypeName> PersistentToLocal(
     v8::Isolate* isolate,
     const v8::Persistent<TypeName>& persistent);
 
+v8::Local<v8::Context> NewContext(
```

Comment:
Some documentation would be helpful so that reader of the code would not need to look it up in node.cc.